Dockerfile.windows

```diff
@@ -166,7 +166,7 @@ FROM microsoft/windowsservercore
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ARG GO_VERSION=1.17.6
-ARG CONTAINERD_VERSION=1.5.9
+ARG CONTAINERD_VERSION=1.6.0-rc.2
```

Comment:
This is probably good for a separate PR (containerd binary should be compatible? if that's not the case, there's a problem)

Reply:
Oh sure yes should be in a dedicated PR, was just testing its behavior with the goto setup.